dev/tools/controllerbuilder/pkg/codegen/typegenerator.go

```diff
@@ -348,6 +348,20 @@ func isAcronym(s string) bool {
 		return true
 	case "http", "https", "ssh":
 		return true
+	case "ip":
+		return true
+	case "gb":
+		return true
+	case "fs":
+		return true
+	case "pd":
+		return true
+	case "kms":
+		return true
+	case "gce":
+		return true
+	case "vtpm":
```

Comment:
Shall this be "vTPM}"?

Reply:
That would mean the variable ends up named `EnablevTPM`, which is not ideal. https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/603f8c85cdd15ab4c605eb1034d22d4d06b4fa14/apis/workstations/v1alpha1/config_types.go#L145

Reply:
Fair enough.